gradle.properties

```diff
@@ -38,3 +38,7 @@ android.nonTransitiveRClass=true
 # https://developer.android.com/build/releases/gradle-plugin#default-changes
 android.defaults.buildfeatures.resvalues=false
 android.defaults.buildfeatures.shaders=false
+
+# Use newer lint version
```

Comment:
nit: we should probably explain the "why" in the comment, wdyt?

```suggestion
# Use newer lint version to support Kotlin 1.9 and corresponding kotlinx-metadata-jvm
```

Reply:
Yes, much better, thanks.